application/views/helpers/Lightgallery.php

```diff
@@ -0,0 +1,114 @@
+                if (strpos($mediaType, 'video') !== false) {
+                    $videoSrcObject = [
+                        'source' => [
+                            [
+                                'src' => $source, 
+                                'type' => $mediaType,
+                            ]
+                        ], 
+                        'attributes' => [
+                            'preload' => false, 
+                            'playsinline' => true, 
+                            'controls' => true,
+                        ],
+                    ];
+                    if (isset($index['tracks'])) {
+                        foreach ($index['tracks'] as $key => $track) {
+                            $label = metadata($track, 'display_title');
+                            $srclang = (metadata($track, 'dcterms:language')) ? metadata($track, 'dcterms:language') : '';
+                            $type = (metadata($track, 'dcterms:type')) ? metadata($track, 'dcterms:type') : 'captions';
+                            $videoSrcObject['tracks'][$key]['src'] = $track->getWebPath();
+                            $videoSrcObject['tracks'][$key]['label'] = $label;
+                            $videoSrcObject['tracks'][$key]['srclang'] = $srclang;
+                            $videoSrcObject['tracks'][$key]['kind'] = $type;
+                        }
+                    }
+                    $videoSrcJson = json_encode($videoSrcObject);
+                    $videoThumbnail = ($media->hasThumbnail()) ? metadata($media, 'thumbnail_uri') : img('fallback-video.png');
+                    $html .= '<div data-video="' . html_escape($videoSrcJson) . '" ' . $mediaCaptionAttribute . 'data-thumb="' . html_escape($videoThumbnail) . '" data-download-url="' . $source . '" class="media resource">';
+                    $html .= '<img src="' . $videoThumbnail . '" alt="" role="presentation">';
+                } else if ($mediaType == 'application/pdf') {
+                    $html .= '<div data-iframe="' . html_escape($source) . '" '. $mediaCaptionAttribute . 'data-src="' . $source . '" data-thumb="' . html_escape(metadata($media, 'thumbnail_uri')) . '" data-download-url="' . $source . '" class="media resource">';
+                    $html .= file_markup($media);
+                } else {
+                    $html .=  '<div data-src="' . $source . '" ' . $mediaCaptionAttribute . 'data-thumb="' . html_escape(metadata($media, 'thumbnail_uri')) . '" data-download-url="' . $source . '" class="media resource">';
+                    $html .= file_markup($media);
+                }
+                $html .= '</div>';
+            }
+            $html .= '</div>';
+        } elseif (isset($sortedMedia['otherMedia'])) {
+            $otherMedia = $sortedMedia['otherMedia'];
+            $html .= '<div id="other-media" class="element">';
+            $html .= '<h3>' . get_view()->translate('Other Media') . '</h3>';
```

Comment:
Inside a helper there's a `$this->view` to get the view the helper was called on: we should be using that instead of the global function here.